synapse/lib/stormtypes.py

```diff
@@ -2112,9 +2114,6 @@ async def wput(self, sha256, url, headers=None, params=None, method='PUT', ssl=T
         if not self.runt.allowed(('axon', 'wput')):
             self.runt.confirm(('storm', 'lib', 'axon', 'wput'))
 
-        if proxy is not None and not self.runt.isAdmin():
```

Comment:
I thought we were moving this to storm.lib.inet.http.proxy as a single permission for controlling proxy bypass?